bin/index_file

```diff
@@ -21,19 +21,13 @@ if [[ ! -z "$2" ]]; then
     logfile=`realpath "$2"`
 fi
 
-
 # Make sure we're using a valid java
 # and get the utility scripts
 source $SCRIPTDIR/set_java_home.sh
 source $SCRIPTDIR/utils.sh
 
-# Put jruby in the path
-export PATH=/htsolr/catalog/bin/jruby/bin:$PATH
-
-
 HOSTNAME=`hostname -s`
-TDIR="/htsolr/catalog-dev/bin/ht_catalog_indexer"
-DDIR=/htsolr/catalog/prep
+TDIR=`realpath $SCRIPTDIR/..`
```

Comment:
@billdueber  `DDIR` wasn't used here; any reason for the hardcoded & apparently-unused `TDIR`?

Reply:
:+1: As we discussed, we can make a list of things to improve/refactor here.